nevergrad/optimization/optimizerlib.py

```diff
@@ -841,21 +840,33 @@ def _internal_provide_recommendation(self) -> tp.ArrayLike:
 class SplitOptimizer(base.Optimizer):
     """Combines optimizers, each of them working on their own variables.
 
-    num_optims: number of optimizers
-    num_vars: number of variable per optimizer.
-    progressive: True if we want to progressively add optimizers during the optimization run.
-    If progressive = True, the optimizer is forced at OptimisticNoisyOnePlusOne.
+    Parameters
+    ---------
+    num_optims: int or None
+        number of optimizers
+    num_vars: int or None
+        number of variable per optimizer.
+    progressive: bool
+        True if we want to progressively add optimizers during the optimization run.
+        If progressive = True, the optimizer is forced at OptimisticNoisyOnePlusOne.
```

Comment:
@teytaud this is the standard way to describe parameters, so that it can be properly rendered on the website